pdk/1.0/modules/ROOT/pages/policies-pdk-architecture.adoc

```diff
@@ -0,0 +1,42 @@
+= Policy Development Kit Architecture Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+The Policy Development Kit (PDK) consists of the:
```

Comment:
```suggestion
Policy Development Kit (PDK) consists of the:
```
I notice you don't include "the" elsewhere. Does this need to be consistent? What is the official branding?

Reply:
https://github.com/mulesoft/docs-gateway/pull/504/files